src/sdk/namespace_fs.js

```diff
@@ -75,8 +80,8 @@ const dir_cache = new LRUCache({
         return { time, stat, sorted_entries, usage };
     },
     validate: async ({ stat }, dir_path) => {
-        const new_stat = await fs.promises.stat(dir_path);
-        return (new_stat.ino === stat.ino && new_stat.mtime.getTime() === stat.mtime.getTime());
+        const new_stat = await nb_native().stat(dir_path);
+        return (new_stat.ino === stat.ino && new_stat.mtime === stat.mtime);
```

Comment:
CAREFUL HERE - this change is wrong now that you changed stat.mtime to be a Date object !
This was a bug I fixed in https://github.com/noobaa/noobaa-core/pull/6394
It would be best to actually compare mtimeMs instead here.
Please fix.

Reply:
Somehow missed leaving the getTime().
I've changed to `mtimeMs`.
Thanks.